### PR TITLE
Fix typos, polish grammar, remove trailing spaces

### PR DESCRIPTION
**Pull request by kevgo**
Thanks for this very cool tool!

- [x] Tests pass
- [x] Appropriate changes to README are included in PR
- [ ] Types updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ let name = 'foo bar'
 await $`mkdir /tmp/${name}`
 ```
 
-Bash is great, but when it comes to writing scripts, 
+Bash is great, but when it comes to writing scripts,
 people usually choose a more convenient programming language.
-JavaScript is a perfect choice, but standard Node.js library 
-requires additional hassle before using. The `zx` package provides
+JavaScript is a perfect choice, but standard Node.js libraries
```
Comment (antonmedv) [inline README.md:23]: It's one library which contains lots of packages.

**Comment by antonmedv** [comment]
Thanks for contributing! ❤️

File: README.md [hunk 2]
```diff
@@ -445,27 +445,27 @@ module.
 
 ### Markdown scripts
 
-The `zx` can execute scripts written in markdown 
+The `zx` can execute scripts embedded in markdown files
```
Comment (antonmedv) [inline README.md:448]: Yes, the zx executes embedded code blocks. But let's think of this as a new syntax. Same as http://go/coffee-script#literate

**Review by antonmedv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kevgo) [reply README.md:23]: Got you, changed to "the Node.js standard library requires additional hassle..."

**Review by antonmedv — APPROVED** [review]

**Pull request merged**